swift-wordle/ContentView.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ContentView.swift
+//  swift-wordle
+//
+//  Created by Kacper Szczepanski on 30/11/2022.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    
+    @ObservedObject var wordleModel: WordleModel
```

Comment:
Można tu od razu utworzyć wordleModel
```suggestion
    @ObservedObject var wordleModel = WordleModel()
```
lub zrobić StateObject w swift_wordleApp i wstrzyknąć go do ContentView  jako .environmentObject, a tutaj @EnvironmentObject - up to you